Code/GraphMol/ChemReactions/testReaction.cpp

```diff
@@ -1304,18 +1301,19 @@ void test12DoubleBondStereochem() {
   TEST_ASSERT(prods[0][0]->getNumBonds() == 5);
   prods[0][0]->updatePropertyCache();
   BOOST_LOG(rdInfoLog) << MolToSmiles(*prods[0][0], true) << std::endl;
-  MolOps::assignStereochemistry(*(prods[0][0]));
   TEST_ASSERT(mol->getBondWithIdx(4)->getStereo() == Bond::STEREOE);
-  TEST_ASSERT(prods[0][0]->getBondWithIdx(3)->getStereo() == Bond::STEREOE);
+
+  Bond *stereoBond = prods[0][0]->getBondWithIdx(3);
+  INT_VECT stereoAtomsRef{{0, 5}};
+  TEST_ASSERT(stereoBond->getStereoAtoms() == stereoAtomsRef);
+  TEST_ASSERT(stereoBond->getStereo() == Bond::STEREOTRANS);
 
   reacts.clear();
 
   // FIX: note that the handling of this situation, where we match double bonds
-  // that
-  // have stereochem indicated, is currently not completely correct since the
-  // stereochem querying stuff doesn't match C/C=C\Cl when C\C=C/Cl is provided
-  // as
-  // a query.
+  // that have stereochem indicated, is currently not completely correct since
```

Comment:
Is there any real reason not to just turn on chirality for the substructure matching?

I really thought we had a while ago, but I think I had been doing this on the outside.  EnumerateLibrary does use subtract matching, but I think it doesn't really reject anything which is a bug there I think.

Reply:
I think turning off the chirality in matching is useful, at least, for debugging.

Reply:
How about adding a `SubstructMatchParameters` data member to the reaction class, then the behavior can be tuned?
https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/Substruct/SubstructMatch.h#L28

Reply:
Hum. Would all the parameters in `SubstructMatchParameters` make sense in a reaction? If not at least most of them, I would stay with the boolean flag.

Reply:
I can see some other things we could add such as sanitize products and such which could be really convenient.

Reply:
At the moment the only two that really apply are `useChirality` and `aromaticMatchesConjugated`, but I am hoping to add a few more of the usual options (special rules for 5-membered heterocycles, etc) for tuning SSS in the near(ish) future (ideally this release cycle). If we used `SubstructMatchParameters` in the reaction code then these would automatically be used there as well. I'm not sure whether or not that's super useful, but it's probably not completely without value. :-)